cornice/tests/test_cors.py

```diff
@@ -228,15 +228,6 @@ def test_resp_dont_include_allow_origin(self):
         self.assertNotIn('Access-Control-Allow-Origin', resp.headers)
         self.assertEqual(resp.json, 'squirels')
 
-    def test_resp_allow_origin_wildcard(self):
-        resp = self.app.options(
-            '/cors_klass',
-            status=200,
-            headers={
-                'Origin': 'lolnet.org',
-                'Access-Control-Request-Method': 'POST'})
-        self.assertEqual(resp.headers['Access-Control-Allow-Origin'], '*')
```

Comment:
Two ways to support removing this test case:

1) it is technically incorrect in CORS to return what this test wants to see i.e. the * should not be returned in AC-Allow-Origin when withCredentials is used.  I found this explicitly stated in Monsur Hossain's book CORS in Action and also in a few StackOverflow threads, most [notably here](http://stackoverflow.com/questions/19743396/cors-cannot-use-wildcard-in-access-control-allow-origin-when-credentials-flag-i/19744754#19744754)

2) The test right after this eliminated test has the same issue that is resolved in the fix.  The AC-Allow-Credentials header is not a request header, but rather a response header, so it shouldn't be set by the client.  When you delete it, you then have 2 tests that expect contradictory results, so one needs to be eliminated anyway.

Make sense?